lib/services/sku-pack-service.js

```diff
@@ -444,28 +462,30 @@ function skuPackServiceFactory(
      * @param  {String}     fromRoot The path to the configuration file
      * @return {Promise}
      */
-    SkuPackService.prototype.validatePack = function(contents, fromRoot) {
-        try {
+    SkuPackService.prototype.validatePack = function(contents, fromRoot, options) {
+        options = options || {};
+
+        return Promise.try(function() {
             var conf = JSON.parse(contents);
```

Comment:
Promise.try() converts synchronous throws into rejects to be caught by .catch().  The error message, along with the notice about the invalid json, occurs at line 487.